app_dart/lib/src/service/luci.dart

```diff
@@ -63,7 +66,23 @@ class LuciService {
   }) async {
     assert(requireTaskName != null);
     final List<LuciBuilder> builders = await LuciBuilder.getProdBuilders(repo, config);
-    final Iterable<Build> builds = await getBuilds(repo, requireTaskName, builders);
+    final List<Build> builds = <Build>[];
+
+    // Get builds data is batches of 50 builders also add retries to the operations.
+    const RetryOptions r = RetryOptions(maxAttempts: 3);
+    for (int j = 0; j < builders.length; j += 50) {
```

Comment:
can we use a variable like `buildBatchSize` instead of fixed 50 here and 49 below?

Reply:
done!